database seed.js file code review

HourglassMagi opened the pull request:
Please review this pull request for me. let me know if there are any significant changes I need to make, including but not limited to adhering to style guide rules. Thank you.

HourglassMagi committed:
created seed data generator for instructions database, corrected a bug in the index.js file where module was mislabled 'model' removed depricated  query.on method from index.js file

HourglassMagi committed:
completed seed file, seeded 100 database records into instructions database recipes table

HourglassMagi commented:
![](https://github.trello.services/images/mini-trello-icon.png) [PR Laurent's database seed branch seed.js file.](https://trello.com/c/W8RkSwUK/36-pr-laurents-database-seed-branch-seedjs-file)

HourglassMagi committed:
modified code to be more within the bounds of styel guide

CareyL96 commented on database/seed.js:
```diff
@@ -0,0 +1,37 @@
+      instructions.instruction = faker.lorem.paragraph();
+      step++;
+      recipe.push(instructions)
+    }
```
looks like this indentation should not be indented

careyl96 commented on database/seed.js:
```diff
@@ -0,0 +1,37 @@
+
+const createFakeData = () => {
+  for(let i = 0; i <= 100; i++) {
+  const steps = Math.floor(Math.random() * 15) + 4;
```
missing indentation in this for loop

careyl96 reviewed (changes requested):
fix indentation issues, add a space after the while loop (I also think it should be a for loop, not a while loop; while loops are generally used to repeat a section of code an unknown number of times), and consider renaming the createFakeData function to insertFakeData.

HourglassMagi committed:
Request Changes

created space after the while loop. Removed unnecessary console logs. added missing semi-colons.

HourglassMagi commented:
Hey @CareyL96 ,
Thanks for the feedback. I made the requested changes. I decided to keep the while loop because I don't know how many steps there will be for a given recipe. They are being randomly generated between 4 and 15 which is why I decided to user a while loop instead of a for loop. Let me know if you see any other changes I need to make. Thanks.

careyl96 reviewed (dismissed):
Missing indentation at the while loop, jsonObj, and text variable initializations since they're all nested within the for loop but you can fix that in the next PR. Consider changing steps to numSteps instead. Looks good otherwise!

HourglassMagi committed:
Merge branch 'master' into seed
